microsite/src/main/tut/advanced/system.md

```diff
@@ -22,6 +22,9 @@ With `currentTime`, you can retrieve the current time as a `java.time.Instant`:
 system.currentTime
 ```
 
+Note: In order to use `currentTime` in a Scala.js project you will need to add the
+[scala-js-java-time](https://github.com/scala-js/scala-js-java-time) dependency to avoid linking errors.
```

Comment:
This dependency got removed here: https://github.com/scalaz/scalaz-zio/commit/a0838c6cdd79dda88b9bd8cc8a0558de24f253f2 because `unusedCompileDependenciesTest` was failing in the build, so just making note of this in the docs for now.

Reply:
We could also add the dependency back and exclude it from the dependency check if that is a better solution.

Reply:
Agreed, let's do that, but merge for now since this is an improvement. 👍